service/history/replication/dlq_handler.go

```diff
@@ -159,11 +159,11 @@ func (r *dlqHandlerImpl) readMessagesWithAckLevel(
 			},
 		)
 		if err != nil {
-			return nil, nil, err
+			return nil, nil, nil, err
 		}
 	}
 
-	return response.ReplicationTasks, resp.NextPageToken, nil
+	return response.ReplicationTasks, taskInfo, resp.NextPageToken, nil
```

Comment:
just want to clarify the behavior here: `taskInfo` will always contain info for all raw tasks, while `response.ReplicationTasks` only contains tasks whose history can be retrieved from the remote cluster?

If that's the case probably in CLI output we should make it more clear that some tasks are missing and urge the user to check raw task info.

Reply:
Added a warning message if the tasks are empty but the metadata task is not empty